conf/dataset.jsonld

```diff
@@ -18,7 +18,7 @@
       },
       "Concept": "http://www.w3.org/2004/02/skos/core#Concept"
    },
-   "id":"https://lobid.org/gnd/dataset#!",
+   "id":"https://rpb.lbz-rlp.de/dataset#!",
```

Comment:
This URI does not seem to work. I think we should just use https://rppd.lobid.org/dataset#! ...

```suggestion
   "id":"https://rppd.lobid.org/dataset#!",
```

Reply:
My bad. I assume that the link will work as soon as this is deployed to production. So please ignore my suggestion.

Reply:
No, you're right, I mixed up RPPD and RPB here!